src/hammer-vlsi/defaults.yml

```diff
@@ -608,3 +619,51 @@ technology.core:
     # Can be a wildcard/glob
     # type: str
     tap_cell_rail_reference: null
+
+sim.inputs:
+  # Top RTL module
+  top_module: null
+  # top_module_config: null
+  input_files: []
+  # Debug option
+  debug: false
+  # Black box command line option escape hatch
+  options: [] 
+  # Define options
+  defines: []
+  # C compiler options
+  compiler_opts: []
+  # Simulation timescale
+  timescale: null
+  # C source files 
+  csrc: []
+  # Benchmarks to run on simulator
+  benchmarks: []
+  # Flags to run with the simulator
+  exec_flags: []
+  # List of TCL scripts to execute in a vcs call
+  tcl_scripts: []
+
+  access_file: ""
+  force_regs_file: ""
+
+  tb_dut: "dut"  
+
+  # Valid options are:
+  #   "rtl" for rtl level simulation
+  #   "gl-syn" for gate level post-synthesis simulation
+  #   "gl-par" for gate level post-par simulation
+  level: "rtl"
+
+  gl-syn:
+    input_files: []
+    post_synth_sdc: null
```

Comment:
can we just call this `sdc`? If it's post-PNR this name doesn't make sense

Reply:
This is removed since it's not actually being used.  I decided to remove the `gl-syn` section because options and input_files inputs can be shared between the different sim levels, and the sim mode is moved to the `level` flag.